charts/authentik/values.yaml

```diff
@@ -118,175 +195,740 @@ authentik:
+    # runAsUser: 1000
+    # runAsGroup: 1000
+    # fsGroup: 1000
+
+  # -- authentik worker container-level security context
+  # @default -- See [values.yaml]
+  containerSecurityContext: {}
+    # Not all of the following has been tested. Use at your own risk.
+    # runAsNonRoot: true
+    # readOnlyRootFilesystem: true
+    # allowPrivilegeEscalation: false
+    # seccomProfile:
+    #   type: RuntimeDefault
+    # capabilities:
+    #   drop:
+    #     - ALL
+
+  # -- terminationGracePeriodSeconds for container lifecycle hook
+  terminationGracePeriodSeconds: 30
+
+  # -- Prority class for the authentik worker pods
+  # @default -- `""` (defaults to global.priorityClassName)
+  priorityClassName: ""
+
+  # -- [Node selector]
+  # @default -- `{}` (defaults to global.nodeSelector)
+  nodeSelector: {}
+
+  # -- [Tolerations] for use with node taints
+  # @default -- `[]` (defaults to global.tolerations)
+  tolerations: []
+
+  # -- Assign custom [affinity] rules to the deployment
+  # @default -- `{}` (defaults to the global.affinity preset)
+  affinity: {}
+
+  # -- Assign custom [TopologySpreadConstraints] rules to the authentik worker
+  # @default -- `[]` (defaults to global.topologySpreadConstraints)
+  ## Ref: https://kubernetes.io/docs/concepts/workloads/pods/pod-topology-spread-constraints/
+  ## If labelSelector is left out, it will default to the labelSelector configuration of the deployment
+  topologySpreadConstraints: []
+    # - maxSkew: 1
+    #   topologyKey: topology.kubernetes.io/zone
+    #   whenUnsatisfiable: DoNotSchedule
+
+  # -- Deployment strategy to be added to the authentik worker Deployment
+  # @default -- `{}` (defaults to global.deploymentStrategy)
+  deploymentStrategy: {}
+    # type: RollingUpdate
+    # rollingUpdate:
+    #   maxSurge: 25%
+    #   maxUnavailable: 25%
+
 
 serviceAccount:
-  # -- Service account is needed for managed outposts
+  # -- Create service account. Needed for managed outposts
   create: true
+  # -- additional service account annotations
   annotations: {}
   serviceAccountSecret:
-    # -- As we use the authentik-remote-cluster chart as subchart, and that chart
-    # creates a service account secret by default which we don't need here, disable its creation
+    # As we use the authentik-remote-cluster chart as subchart, and that chart
+    # creates a service account secret by default which we don't need here,
+    # disable its creation
     enabled: false
-  fullnameOverride: authentik
-  nameOverride: authentik
 
-prometheus:
-  serviceMonitor:
-    create: false
-    interval: 30s
-    scrapeTimeout: 3s
-    # -- labels additional on ServiceMonitor
-    labels: {}
-  rules:
-    create: false
-    # -- labels additional on PrometheusRule
-    labels: {}
 
 geoip:
-  # -- optional GeoIP, deploys a cronjob to download the maxmind database
+  # -- enable GeoIP sidecars for the authentik server and worker pods
   enabled: false
+
+  editionIds: "GeoLite2-City"
```

Comment:
should update this to `GeoLite2-City GeoLite2-ASN` since we have the asn reader merged

Reply:
Yeah, I'll cherry-pick #229